lib/pluginmanager/command.rb

```diff
@@ -1,8 +1,16 @@
 class LogStash::PluginManager::Command < Clamp::Command
+
+  def initialize(invocation_path, context = {}, parent_attribute_values = {})
+    initialize_gemfile
+    super(invocation_path, context, parent_attribute_values)
+  end
+
```

Comment:
I think I know why you aim to have a lazy gemfile, so not to load it until is need. What might be the problems for you to actually load the gemfile once at the "initialization" of the plugin manager?